Code/Framework/AzToolsFramework/AzToolsFramework/Metadata/MetadataManager.cpp

```diff
@@ -91,13 +91,13 @@ namespace AzToolsFramework
             return false;
         }
 
-        outValue = *value;
+        outValue.CopyFrom(*value, document.GetAllocator());
```

Comment:
It probably wont affect anything but it may be worth doing
```
outValue = rapidjson_ly::Document(); 
```

Before calling copyfrom.  Because copyfrom doesn't clear memory.